metrics/src/test/java/com/expedia/adaptivealerting/metrics/functions/source/graphite/GraphiteQueryResultTest.java

```diff
@@ -0,0 +1,81 @@
+package com.expedia.adaptivealerting.metrics.functions.source.graphite;
+
+import lombok.val;
+import org.junit.Test;
+import java.util.Iterator;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class GraphiteQueryResultTest {
+
+    @Test
+    public void testgetGraphiteQueryResultFromJson(){
```

Comment:
Please use the standard Java idiom for method names. Meaning prefer `testGet` to `testget`. This applies throughout.